docs/ui/fragments/angular/configure-module.md

```diff
@@ -0,0 +1,19 @@
+```js
```

Comment:
Why not put `_app.module.ts_` at the top here so you don't have to duplicate it as well? (similar for `main.js` above vue/configure-app.md)

Reply:
Good point, I will do that.